cfbs.json

```
@@ -143,7 +143,7 @@
     },
     "inventory-systemd": {
       "description": "Inventory interesting things from systemd",
-      "tags": ["inventory", "systemd", "experimental"],
+      "tags": ["inventory", "systemd", "supported"],
```

Please put it as the first tag:

```suggestion
      "tags": ["supported", "inventory", "systemd"],
```